specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -3322,6 +3372,24 @@
       },
       "x-ms-discriminator-value": "static"
     },
+    "UnassignJobResponse": {
+      "required": [
+        "jobId",
+        "unassignmentCount"
+      ],
+      "type": "object",
+      "properties": {
+        "jobId": {
+          "description": "The Id of the job unassigned.",
+          "type": "string"
+        },
+        "unassignmentCount": {
+          "format": "int32",
+          "description": "The number of times a job is unassigned. At a maximum 3.",
```

Comment:
If your customers are interested in the max number of times the job can be unassigned then perhaps the max should be returned as an integer?

Reply:
We throw a validation error if unassignmentCount exceeds the maximum.